### PR TITLE
GH Artists

### LINKED ISSUE #605: Review Grasshopper Artists (compas_ghpython)
There are new artists for several objects (Frame, Point, etc) in the ghpython package, and they need to be revised to make sure they match the expected behavior in a GH context.

### PR DESCRIPTION
**Pull request by beverlylytle**
Closes #605 

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [x] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

- [x] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [x] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas_ghpython/artists/circleartist.py [hunk 1]
```diff
@@ -31,12 +31,33 @@ def draw(self):
...
+        Returns
+        -------
+        list of :class:`Rhino.Geometry.Circle`
+
+        """
+        circles = [CircleArtist._get_args(primitive) for primitive in collection]
+        return compas_ghpython.draw_circles(circles)
+
+    @classmethod
+    def _get_args(cls, primitive):
```
Comment (tomvanmele) [inline src/compas_ghpython/artists/circleartist.py]: perhaps this should be `@staticmethod`. no need to pass `cls` as implicit first argument...

File: src/compas_ghpython/artists/frameartist.py [hunk 2]
```diff
@@ -55,22 +57,50 @@ def draw(self):
...
+        args = [FrameArtist._get_args(primitive) for primitive in collection]
+        points, lines = zip(*args)
+        lines = itertools.chain(*lines)
         geometry = [None, None]
         geometry[0] = compas_ghpython.draw_points(points)
         geometry[1] = compas_ghpython.draw_lines(lines)
         return geometry
 
+    @classmethod
+    def _get_args(cls, primitive, scale=1.0, color_origin=(0, 0, 0), color_xaxis=(255, 0, 0), color_yaxis=(0, 255, 0), color_zaxis=(0, 0, 255)):
```
Comment (tomvanmele) [inline src/compas_ghpython/artists/frameartist.py]: `@staticmethod` (same for all others)

**Review by tomvanmele — APPROVED** [review]
small things. otherwise LGTM

**Comment by tomvanmele** [comment]
finished?

**Comment by beverlylytle** [comment]
I don't know if @gonzalocasas wanted to take a look.

**Review by gonzalocasas — APPROVED** [review]
LGTM! I haven't tested it thou, but the code looks good 👍

**Pull request merged**